<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DAMPP</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="plocation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>590</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start_stop_btn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="plocation_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Project Location</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="op_log">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>590</width>
      <height>370</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="op_log_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lhost_btn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>160</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Localhost</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pma_btn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>230</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>PhpMyAdmin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flocation_btn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>300</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>File Location</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>135</y>
      <width>650</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionEdit_Ports"/>
    <addaction name="actionRemove_Services"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDAMPP_Help"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionEdit_Ports">
   <property name="text">
    <string>Edit Ports</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="actionRemove_Services">
   <property name="text">
    <string>Remove Services</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionDAMPP_Help">
   <property name="text">
    <string>DAMPP Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
